### ISSUE TITLE
Add Shapeshifter Role

### ISSUE THREAD

**Issue by MachinePlaygroundPeopleMachine** [scp222thj/MalumMenu#180]
Please add shapeshifter role. 

Edit: Disregard. It pays to read first before posting.

**Comment by aither223** [comment]
how did you change the settings to add the role?

**Comment by MachinePlaygroundPeopleMachine** [comment]
> how did you change the settings to add the role?

I didn't. 

"@everyone 

Among Us was updated yesterday so I will release a new Malum version as soon as possible which is fully compatible. It seems Innersloth might have updated some parts of their anticheat so some Malum features might be broken at the moment. I'll see what I can do but some Malum features might have to be fully removed to prevent users from getting kicked. Because of that, I'll try to add some new features too in this Malum update."

"@everyone 

I have just released Malum v2.4.2, which fixes most issues users were facing with the latest AU version. Update now and it will stop you from getting kicked by the anticheat!

However, I had to move most murder / kill cheats to a new Host-Only tab since Innersloth patched them when playing normal lobbies. You will only be able to access these cheats when hosting lobbies. This is the case will all cheat menus out there, sadly there is no bypass at the moment <:shrug:1259187589684920351> 

However, this update is only a small hotfix and I plan to release another update, titled v2.5.0, which is bigger and will also polish some parts of the menu. Today I simply didn't have time to implement these bigger plans." 

These are two of several of the announcements from their discord. Things like Shapeshifter can't really be added because the anti-cheat was updated, so things like shapeshifter role would need a bypass and I barely know anything about that, let alone how anti-cheat works.
